src/Sulu/Bundle/TestBundle/Resources/dist/sulu.yml

```diff
@@ -89,6 +98,18 @@ doctrine:
                 dir: "%kernel.root_dir%/../../vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity"
                 alias: GedmoTree # this one is optional and will default to the name set for the mapping
                 is_bundle: false
+        dql:
+            string_functions:
+                group_concat: Oro\ORM\Query\AST\Functions\String\GroupConcat
```

Comment:
This was missing from the postegres configuration, I wonder if it was one of the reasons the postgres builds were failing..